oahray/prime-esque
# Add PrimeNumbers class

#### What does this PR do?
This PR will add PrimeNumbers class
#### Description of Task to be completed?
N/A
#### How should this be manually tested?
1. Clone repo or `git pull`
1. cd into project folder and `npm install`.
1. See [readme.md/#Usage](https://github.com/adesege/prime-esque/tree/development#usage) for Usage samples
#### Any background context you want to provide?
N/A
